Makefile

```diff
@@ -23,8 +23,7 @@ depends:
 	  datakit-ci datakit-bridge-github datakit-bridge-local-git -y
 
 datakit:
-	$(BUILD) -n datakit
-	$(RUNTEST) test/datakit
+	$(BUILD) -p datakit
```

Comment:
Why did this go? Are we testing it elsewhere?

Reply:
We are not testing this. And `jbuilder -n datakit` just says that `-n` is not a valid option.